trigger/utils/network.py

```diff
@@ -43,12 +39,10 @@ def ping(host, count=1, timeout=5):
     >>> network.ping('192.168.199.253')
     False
     """
-    # Make the command silent even for bad input
-    if not VALID_HOST_RE.findall(host):
-        return False
 
     ping_command = "ping -q -c%d -W%d %s" % (count, timeout, host)
-    status, results = commands.getstatusoutput(ping_command)
+    #status = subprocess.call(["ping", "-q", "-c", count, "-W", timeout, host], stdout=open('/dev/null','w'),stderr=open('/dev/null','w'))
+    status = subprocess.call(["ping", "-q", "-c", str(count), "-W", str(timeout), host], stdout=open('/dev/null','w'),stderr=open('/dev/null','w'))
```

Comment:
Two main pieces of feedback here:

1. Go ahead and keep `ping_command` intact, but use `shlex.split(ping_command)` when passing it to `subprocess.call()`, like so:

```python
import shlex  # For illustration
subprocess.call(shlex.split(ping_command))
```

2. Instead of opening `/dev/null` directly, use this pattern instead, and add `close_fds=True` to make sure `subprocess.call()` closes any file descriptors:

```python
import os  # For illustration
DEVNULL = open(os.devnull, 'w')
```

The final solution may look something like this:

```python
import os
import shlex

DEVNULL = open(os.devnull, 'w')
status = subprocess.call(
    shlex.split(ping_command), stdout=DEVNULL, stderr=DEVNULL, close_fds=True
)
```